src/app/(app)/users/_components/UsersDialogs.tsx

```
@@ -0,0 +1,110 @@
+        user={editingUser}
+      />
+      {userToDelete && (
+        <AlertDialog
+          open={!!userToDelete}
+          onOpenChange={(open) => !open && onUserToDeleteChange(null)}
+        >
+          <AlertDialogContent>
+            <AlertDialogHeader>
+              <AlertDialogTitle>Xác nhận xóa</AlertDialogTitle>
+              <AlertDialogDescription>
+                Bạn có chắc chắn muốn xóa người dùng "{userToDelete.full_name}"? Hành động này
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)